lib/napa/param_sanitizer/parameter_filter.rb

```diff
@@ -0,0 +1,75 @@
+# The following code is taken from:
+#   https://github.com/rails/rails/blob/master/actionpack/lib/action_dispatch/http/parameter_filter.rb
```

Comment:
Do we make any modifications to this code? I think we pull in `actionpack` in Napa, right? If that's the case, could we just reference this instead of duplicating the code?